### ISSUE TITLE
Optimise the AI story improvement mechanics

### ISSUE THREAD

**Issue by wintermeyer** [wintermeyer/animina#986]
What we are talking about:

<img width="1562" alt="Bildschirmfoto 2024-08-30 um 09 25 45" src="https://github.com/user-attachments/assets/8d68b51d-63f0-4ae4-a583-381470c01b30">

## Thoughts

We should change the GUI and the LLM prompt to make it possible to select multiple options at once and optimise everything in one LLM run. Example: I might want to spell check, make it funnier and shorten it. No need to run it three times through the LLM.

Please use something like this Tailwind HTML code:

    <fieldset>
      <legend class="sr-only">Optimize a Story</legend>
      <div class="space-y-5">
        <div class="relative flex items-start">
          <div class="flex h-6 items-center">
            <input id="comments" aria-describedby="comments-description" name="comments" type="checkbox" class="h-4 w-4 rounded border-gray-300 text-indigo-600 focus:ring-indigo-600">
          </div>
          <div class="ml-3 text-sm leading-6">
            <label for="comments" class="font-medium text-gray-900">Fix spelling and grammar errors.</label>
          </div>
        </div>
        <div class="relative flex items-start">
          <div class="flex h-6 items-center">
            <input id="candidates" aria-describedby="candidates-description" name="candidates" type="checkbox" class="h-4 w-4 rounded border-gray-300 text-indigo-600 focus:ring-indigo-600">
          </div>
          <div class="ml-3 text-sm leading-6">
            <label for="candidates" class="font-medium text-gray-900">Funnier</label>
          </div>
        </div>
        <div class="relative flex items-start">
          <div class="flex h-6 items-center">
            <input id="offers" aria-describedby="offers-description" name="offers" type="checkbox" class="h-4 w-4 rounded border-gray-300 text-indigo-600 focus:ring-indigo-600">
          </div>
          <div class="ml-3 text-sm leading-6">
            <label for="offers" class="font-medium text-gray-900">More Exciting</label>
          </div>
        </div>

        ...
        
      </div>
    </fieldset>

## Important

- Use multicolumn on Desktop. (Use the available horizontal real estate.)
- Spell checking should be on be default. But can be deactivated by the user.
- Think about what makes sense. So shorten and lengthen doesn't make sense to active together. Maybe use a radio button in that case.

## Bottom Line

Make it more useful for the user.
